applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -304,29 +355,48 @@ static void init(void)
 
 	for (size_t i = 0; i < ARRAY_SIZE(subscribers); i++) {
 		bt_gatt_hids_c_init(&subscribers[i].hidc, &params);
+		k_delayed_work_init(&subscribers[i].timeout,
+				    peripheral_timeout_fn);
 	}
 	sys_slist_init(&enqueued_event_list);
 
-	k_delayed_work_init(&config_fwd_timeout, forward_config_timeout);
+	static struct bt_conn_cb conn_callbacks = {
+		.le_param_req = le_param_req,
+	};
+
+	bt_conn_cb_register(&conn_callbacks);
+
+	if (IS_ENABLED(CONFIG_DESKTOP_HID_FORWARD_LOW_LATENCY_ON_HID_INPUT)) {
+		for (size_t i = 0; i < ARRAY_SIZE(rep_id_2_sub_idx); i++) {
+			rep_id_2_sub_idx[i] = INVALID_SUB_IDX;
```

Comment:
I don't think we should make this assumption

Reply:
discussed offline, this will be changed